libkbfs/journal_server.go

```diff
@@ -130,6 +130,9 @@ func makeJournalServer(
 	bcache BlockCache, dirtyBcache DirtyBlockCache, bserver BlockServer,
 	mdOps MDOps, onBranchChange branchChangeListener,
 	onMDFlush mdFlushListener, diskLimiter diskLimiter) *JournalServer {
+	if len(dir) == 0 {
+		panic("dir unexpectedly empty")
```

Comment:
This wording is a bit ambiguos, I first thought it meant the directory has no entries in it.  maybe "dir" -> "dir string"?